### ISSUE TITLE
power tags plant and generator should't be used on the same object

### ISSUE THREAD

**Issue by flacombe** [osm-fr/osmose-backend#470]
Good evening,

According to wiki, power=plant object shouldb't be tagged with any generator:* while any power=generator shouldn't get any plant:* tags.
Is this possible to add a check to prevent any further mistakes with them please?

https://wiki.openstreetmap.org/wiki/Tag:power%3Dplant
https://wiki.openstreetmap.org/wiki/Tag:power%3Dgenerator

All the best

**Comment by frodrigo** [comment]
- 20% of power=plant have a generator:source=*
- no plant:* on power=generator

20% is an hight rate for a forbidden combination. Should we really raise issues for a so common case ?

**Comment by flacombe** [comment]
Yes we should since it confuses devices properties on a facility feature.
plant shouldn't mix with generators.

People get the habit to add generator:* on plants since there was no tags plant:* to propagate the hosted generators properties to the plant.
Then osmose should warn about this and propose a fix with plant:* tags
